ex5.1_this_is_the_way.py

```diff
@@ -0,0 +1,12 @@
+import os
+INPUT_FOLDER_MESSAGE="insert the path to the folder:"
+# function that print the file in folder that start with "deep"
+def print_deep_folder():
+    folder_path = input(INPUT_FOLDER_MESSAGE)
```

Comment:
receive this as a function parameter and access a constant from the outside - it makes you code coupled with a specific path